Week2/homework/map_filter.js

```diff
@@ -3,3 +3,14 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+
+// const newNumbers = [];
+// for (let i = 0; i < numbers.length; i++) {
+//     if (numbers[i] % 2 !== 0) {
+//       newNumbers.push(numbers[i] * 2);
+//     }
+//   }
+//   console.log(newNumbers);
+
+const newNum = numbers.filter(numbers => numbers % 2 !== 0).map(numbers => numbers * 2);
+console.log(newNum);
```

Comment:
Hi @SignosRk, I would recommend that when you think your code is ready to be pushed, remove all the `console.log` statements you might have added. It helps you as a developer primarily as a debugging tool but need not be pushed. 🙂